src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.skia.cs

```diff
@@ -3211,6 +3213,71 @@ public async Task When_FeatureConfiguration_Changes()
 			Assert.AreEqual(1, SUT.TextBoxView.DisplayBlock.Opacity);
 		}
 
+		[TestMethod]
+		public async Task When_Caret_Color_DarkMode()
+		{
+			var useOverlay = FeatureConfiguration.TextBox.UseOverlayOnSkia;
+			using var _1 = Disposable.Create(() => FeatureConfiguration.TextBox.UseOverlayOnSkia = useOverlay);
+
+			// The TextBox is purposefully empty. We want the only content pixels to come from the caret.
+			var SUT = new TextBox
+			{
+				Width = 150
+			};
+
+			await UITestHelper.Load(SUT);
+
+			SUT.Focus(FocusState.Programmatic);
+			await WindowHelper.WaitForIdle();
+
+			var screenshot = await UITestHelper.ScreenShot(SUT);
+			var i = 0;
+			for (; i < 20; i++)
+			{
+				await Task.Delay(100);
```

Comment:
Let's wait using a random delay so we don't align with the blinking unwillingly.